server/src/entity/Absence.ts

```diff
@@ -45,6 +46,9 @@ export class Absence extends BaseEntity {
     @Field()
     exam: boolean
 
+    @Column()
+    studentId: string
```

Comment:
Where is the studentId set? Shouldn't it be set in the create mutation or is there an automation?

Reply:
TypeORM will automagically assign this value with the foreign key belonging to the student relation and will keep it in sync accordingly.